modules/accessing-the-administrator-perspective.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// web_console/about-administrator-perspective.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-the-administrator-perspective_{context}"]
+= Accessing the Administrator perspective
+
+The *Administrator* perspective in the {product-title} web console provides workflows specific to administrator use cases. You can access the *Administrator* perspective from the web console as follows:
+
+.Prerequisites
+To access the *Administrator* perspective, ensure that you have logged in to the web console.
+
+.Procedure
+
+. Log in to the {product-title} web console using your login credentials.
+
+. Use the menu bar to display available menu items and applications for *Administrators*.
```

Comment:
are these steps telling user how to access `Administrator` perspective? they look less relevant ~

Reply:
I would also note that both "Accessing" modules have log in as a prerequisite, and then logging in as a first step. It should be one or the other, not both.